graspy/align/orthogonal_procrustes.py

```diff
@@ -0,0 +1,113 @@
+# Copyright (c) Microsoft Corporation and contributors.
+# Licensed under the MIT License.
+
+from scipy.linalg import orthogonal_procrustes
+
+from .base import BaseAlign
+
+
+class OrthogonalProcrustes(BaseAlign):
+    """
+    Computes the matrix solution of the classical orthogonal Procrustes [1]_
+    problem, which is that given two matrices `X` and `Y` of equal shape
+    (n, d), find an orthogonal matrix `self.Q_` that most closely maps `X` to
+    `Y`. Subsequently, uses that matrix to transform either the original `X`,
+    or a different dataset that has the same generating distribution.
```

Comment:
```suggestion
    or a different dataset in the same space.
```

Reply:
okay, i am happy to incorporate this change, because my comment is probably more confusing, and not exactly correct. the reason why i specified it is: it only makes sense to pass X to transform that has the same orientation as X passed to fit (for example, if they were omni embedded, or are from the same sample).

Reply:
changed.